### ISSUE TITLE
[a11y-defect-2]: Benefit Letters page must manage keyboard focus on first render (page load)

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#96329]
### Point of contact

Trevor Pierce

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

The logged in [Benefits letters confirm address page](https://staging.va.gov/records/download-va-letters/letters/confirm-address) is not managing focus to the H1 correctly on first page load. This is confusing to screen readers because either no information is being read or too much information is being read. Ideally we want to hear "VA Letters and Documents, heading level one" and nothing else when the page loads. Screenshot attached below.

---
![Image](https://github.com/user-attachments/assets/f14041f7-4a78-4807-ad69-025327e53c5c)

### Reproduction steps

1. Log into staging with a test user who has benefit letters. I had good luck with user 54.
2. Turn on the screen reader of your choice
3. Navigate to https://staging.va.gov/records/download-va-letters/
4. Click the link to download benefit letters (it's the primary action link with the green arrow next to it)
5. Verify you do not hear the H1 read aloud
6. You can also programmatically verify focus by opening dev tools and typing `document.activeElement` into the console.

### Proposed solution or next steps

We should add a React `ref` and `tabindex="-1"` to the H1. Using the `useEffect` hook we can then check for the ref and set focus on the H1 on first render.

### References, articles, or WCAG support

1. Understanding [SC 1.3.2](https://w3.org/TR/WCAG22#meaningful-sequence): Meaningful Sequence (Level A)

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [x] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [x] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
